src/components/LoadingBar/LoadingBar.js

```diff
@@ -27,8 +26,10 @@ const wrapperStyles = ({ theme }) => css`
 const progressStyles = ({ theme, size, value, max }) => {
   const outerBorderWidth = '1px';
   const outerBorderRadius = theme.borderRadius.mega;
-  const innerBorderRadius = `${stripUnit(outerBorderRadius) -
-    stripUnit(outerBorderWidth)}px`;
+  const innerBorderRadius = `${subtractUnit(
```

Comment:
While I'm all for these neat helper functions, I wonder if this is really necessary. What I've done in these cases is to use `calc`. That should be much more efficient. @mlent mentioned that ancient android does not support `calc`, but I guess in particular in this case, that would not really matter.